src/scripts/genLttngProvider.py

```diff
@@ -700,48 +709,43 @@ def generateLttngFiles(etwmanifest,eventprovider_directory):
 #define PAL_realloc realloc
 #include "pal/stackstring.hpp"
 """)
-        lTTngImpl.write("#include \"" + lttngevntheadershortname + "\"\n\n")
+            lttngimpl_file.write("#include \"" + lttngevntheadershortname + "\"\n\n")
```

Comment:
Let's make sure that LTTng support continues to work with this change, since the change modifies the LTTng tracepoint generation code.